frontend/src/utils.js

```diff
@@ -250,9 +252,23 @@ export const normaliseText = (name) => {
 }
 
 export const applicableDiagnosticRules = (canteen) => {
+  let bioThreshold = 20
+  let qualityThreshold = 50
+  // group1 : guadeloupe, martinique, guyane, la_reunion, TODO saint_martin
+  const group1 = ["01", "02", "03", "04"]
+  if (group1.indexOf(canteen.region) > -1) {
+    bioThreshold = 5
+    qualityThreshold = 20
+  } else if (canteen.region == "06") {
```

Comment:
```suggestion
  } else if (canteen.region === "06") {
```

La comparaison simple est faite exprès ? (Sachant que plus haut avec le `indexOf` on a une comparaison de type aussi)

```javascript
6 == "06" // true
6 === "06" // false
```